### ISSUE TITLE
Typst: Strong emphasis function cannot take multiline variable

### ISSUE THREAD

**Issue by axelkar** [jgm/pandoc#9807]
**Explain the problem.**

When trying to pass a multiline (so not `#let foo = [bar baz]`) content variable to the [strong emphasis function](https://typst.app/docs/reference/model/strong/), pandoc errors while typst does not.

```sh
pandoc --from typst --to html5 --no-highlight
```
```typst
#let foo = [
  bar baz
]
*#foo*
// Also this:
//#strong(foo)
```
It errors with the following error:
```
(line 1, column 2): unexpected Elt {eltName = Identifier "parbreak", eltPos = Just (line 1, column 2),
eltFields = fromList []} or Elt {eltName = Identifier "strong", eltPos = Just (line 1, column 2), eltFields
= fromList [(Identifier "body",VContent (fromList [Elt {eltName = Identifier "text", eltPos = Just (line 1,
column 2), eltFields = fromList [(Identifier "body",VContent (fromList [Txt "\nbar baz"]))]},Elt {eltName
= Identifier "parbreak", eltPos = Just (line 1, column 2), eltFields = fromList []}]))]}
expecting end of input
Pattern match failure in 'do' block at src/Text/Pandoc/Readers/Typst.hs:108:26-33
```

**Pandoc version?**
pandoc version 3.2 on https://pandoc.org/try

**Comment by jgm** [comment]
It's not entirely clear how we should handle this (though certainly we shouldn't have a pattern match failure!).

Pandoc's Strong element can only contain inline content, and a typst block could be e.g. several paragraphs and a bullet list.  That just can't go inside a Strong.  I suppose we could add Strong to the inline contents of these things.

**Comment by jgm** [comment]
Closed by 372cd1d3b19b94ce7a8a72349a5fb6d735025878
